database/migrations/2021_08_16_170943_delete_columns_removed_from_blade_forms.php

```diff
@@ -13,10 +13,6 @@ class DeleteColumnsRemovedFromBladeForms extends Migration
      */
     public function up()
     {
-        Schema::table('category_terms', function (Blueprint $table) {
-            $table->dropColumn(['local_title', 'local_subtype']);
-        });
-
```

Comment:
As a general rule old migrations should not be modified in order to make new changes/updates to the DB. 

When we run `php artisan migrate` it will only run new migration since the last run. So in this case, it looks like the intention is "don't drop these two columns." Deleting the drop command from this old migration won't bring these two columns back into our DB structures in our dev, staging and prod environments where we already have the app set up. This change will be ignored since it lives in a migration that's older than our last run.

Instead, leave this migration alone, and create a new migration that adds these columns back.

Reply:
@nikhiltri gotcha! I didn't know that was what happened when we deployed. I've reverted the change on that old migration.